levenshtein/dart/code.dart

```diff
@@ -40,20 +47,20 @@ void main(List<String> args) {
   int times = 0;
   // Compare each pair of arguments exactly once
   for (int i = 0; i < args.length; i++) {
-      for (int j = i+1; j < args.length; j++) {
-        if (i != j) {
-            int distance = levenshteinDistance(args[i], args[j]);
-            if (minDistance == -1 || distance < minDistance ) {
-                minDistance = distance;
-            }
-            times++;
+    for (int j = i + 1; j < args.length; j++) {
```

Comment:
The Dart implementation fails the check for correctness:

```
❯ ../run.sh check  
...
Checking Dart
Incorrect output:
times: 1953
min_distance: 7
Check failed for Dart.
...
```

This is not introduced by your code, @vincevargadev. But you could include the fix anyway. Like so:

```dart
  // Compare each pair of arguments twice (i, j) and (j, i), to adhere to the reference implementations
  for (int i = 0; i < args.length; i++) {
    for (int j = 0; j < args.length; j++) {
```

Here the suggestion is to make it clear that we do the redundant check on purpose, so that people will not send PRs “fixing” it. 😄